src/state/control.ts

```diff
@@ -7,33 +7,43 @@ interface ControlStateSchema {
   states: {
     arm: {}
     flipper: {}
+    nothing: {}
   }
 }
 
 type ControlEvent =
   | { type: 'CONTROL_ARM' }
   | { type: 'CONTROL_FLIPPER' }
+  | { type: 'DISABLE' }
   | { type: 'TOGGLE' }
 
 export const controlMachine = Machine<
   ControlContext,
   ControlStateSchema,
   ControlEvent
 >({
-  id: 'gamepad',
+  id: 'control',
   initial: 'flipper',
   context: {},
   states: {
     arm: {
       on: {
         CONTROL_FLIPPER: 'flipper',
         TOGGLE: 'flipper',
+        DISABLE: 'nothing',
       },
     },
     flipper: {
       on: {
         CONTROL_ARM: 'arm',
```

Comment:
Why are there two enabling mechanisms? (toggle + the CONTROL_ARM)

Reply:
Because sometimes you don't know which mode you (you as in the code calling it) are in you just want to toggle it. It allows you to toggle between flipper and arm but you can also go to arm or flipper from the nothing state